### PR TITLE
Node-Health-Check and Self-Node-Remediation operators deployment

### PR DESCRIPTION
**Pull request by ebattat**
## Type of change
Note: Fill **x** in []
- [ ] bug
- [X] enhancement
- [ ] documentation
- [ ] dependencies

## Description

Deployment of Node Health Check and Self Node Remediation Operators.
These operators are intended to assist in tracking VM migration during OCP upgrades and chaos testing

### CODE REVIEW

File: benchmark_runner/common/ocp_resources/create_nhc_snr.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+                # verify once after create all resource files
+                self.wait_for_ocp_resource_create(operator='nhc',
+                                                  verify_cmd="oc get csv -n openshift-workload-availability -o jsonpath='{.items[0].status.phase}'",
+                                                  status='Succeeded')
+                self.wait_for_ocp_resource_create(operator='snr',
+                                                  verify_cmd="oc get csv -n openshift-workload-availability -o jsonpath='{.items[1].status.phase}'",
+                                                  status='Succeeded')
+            if '04_nhc_snr.yaml' in resource:
+                # Verify NHC is enabled
+                self.wait_for_ocp_resource_create(operator='nhc',
```
Comment (RobertKrawitz) [inline benchmark_runner/common/ocp_resources/create_nhc_snr.py:39]: Do you also need to enable the self-node-remediation operator?

Comment (ebattat) [reply benchmark_runner/common/ocp_resources/create_nhc_snr.py:39]: Self-Node-Remediation is auto enabled when Node-Health-Check is enabled

Comment (RobertKrawitz) [reply benchmark_runner/common/ocp_resources/create_nhc_snr.py:39]: Just add a comment here that it's not necessary to automatically enable SNR and I'll approve it.

**Review by RobertKrawitz — CHANGES_REQUESTED** [review]

**Comment by ebattat** [comment]
@RobertKrawitz ,any more comments ?

**Review by RobertKrawitz — APPROVED** [review]

**Pull request merged**